### PR TITLE
Post on keyboard shortcuts for menu items

### PR DESCRIPTION
**Pull request by pireddag**
An article on shortcuts for menu items

### CODE REVIEW

**Comment by pireddag** [comment]
notes.atom has many changes that correspond to file modification dates of files I did not change. Please let me know if I should "resynchronize" my files with the ones of the upstream repo or if we can leave things this way.

File: src/menu-shortcuts.tm [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+  <\scm-code>
+    (kbd-map ("A-e f"\ 
+
+    (make 'footnote)))
+  </scm-code>
+
+  in <shell|my-init-texmacs.scm> I obtained a shortcut for footnotes.
+
+  <paragraph|Searching for text in TeXMacs Scheme files>
```
Comment (da-liii) [inline src/menu-shortcuts.tm]: typo: TeXMacs

**Review by da-liii — APPROVED** [review]
LGTM

**Comment by pireddag** [comment]
I have corrected the typo, fixed something else in my text (already forgot what) and also regenerated the website, I hope it makes sense.

**Comment by mgubi** [comment]
> notes.atom has many changes that correspond to file modification dates of files I did not change. Please let me know if I should "resynchronize" my files with the ones of the upstream repo or if we can leave things this way.

This is a standing issue with the atom feed. I do not know how reliably determine the modification date of a document, maybe from the last logged change in the git history, or something like this. modified dates of git-managed files seems not to be relevant. Suggestions? @darcy-shen @woutersj @pireddag @mdbenito ???

**Comment by pireddag** [comment]
> This is a standing issue with the atom feed. I do not know how reliably determine the modification date of a document, maybe from the last logged change in the git history, or something like this. modified dates of git-managed files seems not to be relevant. Suggestions? @darcy-shen @woutersj @pireddag @mdbenito ???

For the moment no suggestions, as I don't know git well enough to find a way, but I'll keep the issue in mind.

**Comment by da-liii** [comment]
I suggest that you should remove the atom part in this pull request. We could write a CI to improve the workflow.

**Comment by pireddag** [comment]
> I suggest that you should remove the atom part in this pull request.

I think I removed it.

**Pull request merged**